README.md

```diff
@@ -61,6 +61,8 @@ For more information and detailed documentation about the API visit https://deve
 
 This gem requires Ruby 2.4 as of version 7.0.
 
+**Note**: when we release the next major version, we will be dropping support for Ruby 2.4 and 2.5, which have now reached EOL, as per [the releases page](https://www.ruby-lang.org/en/downloads/branches/).
```

Comment:
I added it as a `puts` in `lib/shopify_api.rb` because I don't think we can assume any logging facility will be available at that point, but lmk if anyone can see a better way of doing that!